grove/measurements/estimation.py

```diff
@@ -180,6 +189,7 @@ def estimate_pauli_sum(pauli_terms, basis_transform_dict, program,
         sample_variance = coeff_vec.T.dot(covariance_mat).dot(coeff_vec) / results.shape[1]
 
     return EstimationResult(expected_value=coeff_vec.T.dot(np.mean(results, axis=1)),
+                            pauli_expectations=np.multiply(coeff_vec, np.mean(results, axis=1)),
```

Comment:
HAHAHA, sorry @ncrubin, I think I wanted either a name change or multiplying by the coeffs, but I am not sure I like it with both changes. I am leaning towards the current name but without multiplying. We can document that
`expected_value == coef_vec.dot(pauli_expectations)`.

Reply:
@ntezak I think if we wanted to call it `pauli_moments` then we could return the expected values of the bare Pauli operators (no coeff multiplication).  But keeping `pauli_expectations` would be consistence with multiplying the moment expectation by the coeff.  I like it this way since it is the same "type" as the full expected value--i.e. a sum of coeff * moment expectation.

I vote we keep as is.  I can document update the docs for `expected_value`

Reply:
Ok, sounds good